index.js

```diff
@@ -0,0 +1,79 @@
+console.log("Hello world");
+//task one
+let yearOfBirth = 1989;
+let yearFuture=2027;
+let age = yearFuture - yearOfBirth;
```

Comment:
I know the teacher didn't emphasize it, but it's preferred to set variables as `const` if they are not to be reassigned later. A good way to remember is "Always use `const` until it doesn't `let` you to."